unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/BackpackEditorV2Plugin.cs

```diff
@@ -0,0 +1,19 @@
+namespace DCL.Backpack
+{
+    public class BackpackEditorV2Plugin : IPlugin
+    {
+        private readonly BackpackEditorHUDController hudController;
+
+        public BackpackEditorV2Plugin()
+        {
+            var view = BackpackEditorHUDV2ComponentView.Create();
+            hudController = new BackpackEditorHUDController(view, DataStore.i);
+            view.Initialize();
```

Comment:
Shouldnt the view either be initialized by the controller or before the injection?

Reply:
This was a change introduced by me... and yeah, you're right, the view initialization should be managed by the own controller.

I have commited the change here: https://github.com/decentraland/unity-renderer/pull/4916/commits/057c8f52ae373adc0f6f2ed92165d7c1ae7b88a8

Thanks!